modules/oxide-components/src/main/ts/components/contexttoolbar/ContextToolbar.stories.tsx

```diff
@@ -0,0 +1,343 @@
+              <IconButton icon='arrow-up' onClick={fn()} resolver={Fun.constant(resolvedIcon)} />
+              <span style={{
+                padding: '8px',
+                fontSize: '12px',
+                flexShrink: 0,
+                whiteSpace: 'nowrap'
+              }}>
+                1/3
+              </span>
+              <IconButton icon='arrow-down' onClick={fn()} resolver={Fun.constant(resolvedIcon)} />
+            </ContextToolbar.Group>
+            <ContextToolbar.Group>
+              <Button variant='primary' onClick={fn()}>Accept</Button>
+              <Button variant='secondary' onClick={fn()}>Reject</Button>
+            </ContextToolbar.Group>
+          </ContextToolbar.Toolbar>
+        </ContextToolbar.Root>
+      </div>
+    );
+  }
+};
+
+export const Corners: Story = {
+  parameters: {
+    docs: {
+      description: {
+        story: `Demonstrates CSS anchor positioning with 9 trigger buttons placed at different positions.`,
+      },
+    },
+  },
+  render: () => {
+    const triggerPositions = useMemo(() => ([
+      // Top row
+      { id: 'top-left', label: 'Top Left', style: { top: '20px', left: '20px' }},
+      { id: 'top-center', label: 'Top Center', style: { top: '20px', left: '50%', marginLeft: 'calc(-1 * (6ch + 24px) / 2)' }},
+      { id: 'top-right', label: 'Top Right', style: { top: '20px', right: '20px' }},
+      // Middle row
+      { id: 'middle-left', label: 'Middle Left', style: { top: '50%', left: '20px', marginTop: 'calc(-1em / 2)' }},
+      { id: 'center', label: 'Center', style: { top: '50%', left: '50%', marginTop: 'calc(-1em / 2)', marginLeft: 'calc(-1 * (4ch + 24px) / 2)' }},
+      { id: 'middle-right', label: 'Middle Right', style: { top: '50%', right: '20px', marginTop: 'calc(-1em / 2)' }},
+      // Bottom row
+      { id: 'bottom-left', label: 'Bottom Left', style: { bottom: '20px', left: '20px' }},
+      { id: 'bottom-center', label: 'Bottom Center', style: { bottom: '20px', left: '50%', marginLeft: 'calc(-1 * (9ch + 24px) / 2)' }},
+      { id: 'bottom-right', label: 'Bottom Right', style: { bottom: '20px', right: '20px' }}
+    ] as const), []);
+
+    return (
+      <div className='tox context-toolbar-anchors' style={{ width: '520px' }}>
+        <div
+          className='tox'
+          style={{
+            position: 'relative',
+            height: '360px',
+            border: '1px solid #c6cdd6',
+            borderRadius: '16px',
+            background: '#ffffff',
+            overflow: 'hidden'
+
+          }}
+        >
+          {triggerPositions.map((pos) => (
+            <ContextToolbar.Root key={pos.id} persistent={false}>
+              <ContextToolbar.Trigger>
+                <div style={{ position: 'absolute', display: 'inline-flex', ...pos.style }}>
+                  <Button>{pos.label}</Button>
+                </div>
+              </ContextToolbar.Trigger>
+              <ContextToolbar.Toolbar>
+                <ContextToolbar.Group>
```

Comment:
Would we at this point add support for name and labels for the groups or is that scope creep. https://www.tiny.cloud/docs/tinymce/latest/contexttoolbar/#add-labels-and-groups-for-context-toolbar-buttons